packages/dd-trace/test/appsec/waf/index.spec.js

```diff
@@ -252,14 +263,14 @@ describe('WAF Manager', () => {
 
         wafContextWrapper.run(params)
 
-        expect(Reporter.reportAttack).to.be.calledOnceWithExactly(result.data)
+        expect(Reporter.reportAttack).to.be.calledOnceWithExactly(JSON.stringify(result.events))
```

Comment:
I think it's better to hardcode values most of the time:
1. easier to read
2. less room for errors like false positives/negatives

```suggestion
        expect(Reporter.reportAttack).to.be.calledOnceWithExactly('["ATTACK DATA"]')
```